akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala

```diff
@@ -215,6 +217,11 @@ object HttpEntity {
     if (contentType == Empty.contentType) Empty
     else HttpEntity.Strict(contentType, data = ByteString.empty)
 
+  JavaInitialization.initializeStaticFieldWith(
+    Empty,
+    classOf[jm.HttpEntity].getField("EMPTY"),
+    classOf[jm.HttpEntities].getField("EMPTY"))
```

Comment:
Are these secondary targets really needed? I thought that with the placement of the constants in separate classes the problem should be fixed.